opba-embedded-starter/src/main/resources/application.yml

```diff
@@ -2,7 +2,7 @@ server:
   port: 8085
 spring:
   datasource:
-    url: jdbc:postgresql://localhost:15432/open_banking?currentSchema=banking_protocol
+    url: jdbc:postgresql://localhost:5432/open_banking?currentSchema=banking_protocol
```

Comment:
This is a standard postgres port that may overlap with postgres of sandbox run on the same machine. Therefore we changed it to another one.

Reply:
application.yml is a roadmap for developers who start with the project, why hardcoding customary non-default value if it can be easily overridden using SPRING_DATASOURCE_URL env. variable or property
Postgres can have multiple schemas(databases) for each application